package.azure-pipelines.yml

```diff
@@ -42,7 +42,6 @@ jobs:
     inputs:
       extensionVersion: '$(Extension.Version)'
       updateTasksVersion: true
-      updateTasksVersionType: patch
```

Comment:
I believe this is no longer needed with the latest version of the `PackageAzureDevOpsExtension` and is causing a build failure.